src/utils/default-config.ts

```diff
@@ -0,0 +1,42 @@
+import type { BootstrapInit } from '@libp2p/bootstrap'
+import type { AddressManagerInit, ConnectionManagerInit } from 'libp2p'
+
+export interface BootstrapConfig {
+  addresses: Omit<AddressManagerInit, 'announceFilter'>
+  connectionManager: ConnectionManagerInit
+  bootstrap: BootstrapInit
+  identity: {
+    peerId: string
+    privKey: string
+  }
+}
+
+export const defaultConfig: BootstrapConfig = {
+  addresses: {
+    announce: [],
+    noAnnounce: [],
+    listen: [
+      '/ip4/0.0.0.0/tcp/4003/ws',
+      '/ip4/0.0.0.0/tcp/4001',
+      '/ip6/::/tcp/4004/ws',
+      '/ip6/::/tcp/4001',
+      '/ip4/0.0.0.0/udp/4002/webrtc'
+    ]
+  },
+  bootstrap: {
+    list: [
+      '/dns4/am6.bootstrap.libp2p.io/tcp/443/wss/p2p/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb',
+      '/dns4/sg1.bootstrap.libp2p.io/p2p/QmcZf59bWwK5XFi76CZX8cbJ4BhTzzA3gU1ZjYZcYW3dwt',
+      '/dns4/sv15.bootstrap.libp2p.io/p2p/QmNnooDu7bfjPFoTZYxMNLWUQJyrVwtbZg5gBMjTezGAJN'
+    ]
+  },
+  identity: {
+    peerId: 'generated by `auto-config.ts`',
+    privKey: 'generated by `auto-config.ts`'
+  },
```

Comment:
we should remove peerId, since we derive that from privateKey. We should rename privKey to privateKey

Reply:
Resolved in https://github.com/libp2p/js-libp2p-amino-dht-bootstrapper/pull/151/commits/c7906126be93ca4ffdadff58158d57d2f0cfc640